### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DandyDeveloper**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# reddit-bot
\ No newline at end of file
+# Reddit autopost for up-for-grabs > /r/RespositoryRequest
```
Comment (dahlbyk) [inline README.md]: Typo: Re<b>s</b>pository

Maybe link to https://www.reddit.com/r/RepositoryRequest, too?

Comment (DandyDeveloper) [reply README.md]: Updated. Sorry.

File: README.md [hunk 2]
```diff
@@ -1 +1,4 @@
-# reddit-bot
\ No newline at end of file
+# Reddit autopost for up-for-grabs > /r/RepositoryRequest
+
+
+Http://www.reddit.com/r/RepositoryRequest
```
Comment (dahlbyk) [inline README.md]: Reddit always redirects to `https`, so we might as well use that in our link.

Comment (DandyDeveloper) [reply README.md]: I'm doing well today. Update submitted.

**Comment by dahlbyk** [comment]
:clap:

**Pull request merged**